docs/_data/errorref/kuma_errors_export.json

```diff
@@ -0,0 +1,134 @@
+{
+    "errors_export": {
+        "component_name": "kuma",
+        "component_type": "adapter",
+        "errors": {
+            "1000": {
+                "name": "ErrInstallKumaCode",
+                "code": "1000",
+                "severity": "none",

```

Comment:
as this page will be published, I suggest to remove all the dummy data from the input files, i.e. all `severity` , `long_description`, `short_description`, `probable_cause`, `suggested_remediation`.  (I probably shouldn't have included it, sorry).

Reply:
Food for thought: edits and commits can be made directly in GitHub's web interface by pressing CTL+G (or CMD+G). I mention this as it might move along part of this process more quickly.